content/en/docs/examples/microservices-istio/setup-kubernetes-cluster/index.md

```diff
@@ -0,0 +1,228 @@
+      resources: ["*"]
+      verbs: ["*"]
+    ---
+    kind: RoleBinding
+    apiVersion: rbac.authorization.k8s.io/v1beta1
+    metadata:
+      name: ${NAMESPACE}-access
+      namespace: $NAMESPACE
+    subjects:
+    - kind: ServiceAccount
+      name: ${NAMESPACE}-user
+      namespace: $NAMESPACE
+    roleRef:
+      apiGroup: rbac.authorization.k8s.io
+      kind: Role
+      name: ${NAMESPACE}-access
+    ---
+    kind: RoleBinding
+    apiVersion: rbac.authorization.k8s.io/v1beta1
+    metadata:
+      name: ${NAMESPACE}-istio-system-access
+      namespace: istio-system
+    subjects:
+    - kind: ServiceAccount
+      name: ${NAMESPACE}-user
+      namespace: $NAMESPACE
+    roleRef:
+      apiGroup: rbac.authorization.k8s.io
+      kind: Role
+      name: istio-system-access
+    EOF
+    {{< /text >}}
+
+1.  Each participant needs to use their own Kubernetes configuration file. This configuration file specifies
+    the cluster details, the service account, the credentials and the namespace of the participant.
+    The `kubectl` command uses the configuration file to operate on the cluster.
+
+    Generate a Kubernetes configuration file for each participant:
+
+    {{< text bash >}}
+    $ cat <<EOF > ./${NAMESPACE}-user-config.yaml
+    apiVersion: v1
+    kind: Config
+    preferences: {}
+
+    clusters:
+    - cluster:
+        certificate-authority-data: $(kubectl get secret $(kubectl get sa ${NAMESPACE}-user -n $NAMESPACE -o jsonpath={.secrets..name}) -n $NAMESPACE -o jsonpath='{.data.ca\.crt}')
+        server: $(kubectl config view -o jsonpath="{.clusters[?(.name==\"$(kubectl config view -o jsonpath="{.contexts[?(.name==\"$(kubectl config current-context)\")].context.cluster}")\")].cluster.server}")
+      name: ${NAMESPACE}-cluster
+
+    users:
+    - name: ${NAMESPACE}-user
+      user:
+        as-user-extra: {}
+        client-key-data: $(kubectl get secret $(kubectl get sa ${NAMESPACE}-user -n $NAMESPACE -o jsonpath={.secrets..name}) -n $NAMESPACE -o jsonpath='{.data.ca\.crt}')
+        token: $(kubectl get secret $(kubectl get sa ${NAMESPACE}-user -n $NAMESPACE -o jsonpath={.secrets..name}) -n $NAMESPACE -o jsonpath={.data.token} | base64 --decode)
+
+    contexts:
+    - context:
+        cluster: ${NAMESPACE}-cluster
+        namespace: ${NAMESPACE}
+        user: ${NAMESPACE}-user
+      name: ${NAMESPACE}
+
+    current-context: ${NAMESPACE}
+    EOF
+    {{< /text >}}
+
+1.  If you are setting up the cluster for yourself, copy the
+    `${NAMESPACE}-user-config.yaml` file mentioned in the previous steps to your
+    local computer, where `${NAMESPACE}` is the name of the namespace you
+    provided in the previous steps. For example, `tutorial-user-config.yaml`.
+    You will need this file later in the tutorial.
+
+    If you are an instructor, send the generated configuration files to each
+    participant who should copy it to their local computer.
+
+Congratulations, you created your cluster!
```

Comment:
"You created your cluster" -> "you configured your cluster for the tutorial", maybe also "... and now you can start actual learning".